### ISSUE TITLE
SSLstrip dosen't remove https

### ISSUE THREAD

**Issue by Phantom852** [sensepost/mana#31]
i've installed mana toolkit since a while now, and till this i wasn't able to sslstrip any of https websites, i've tried to look up articles from everywhere, but nothing. 

i'll appreciate any help

**Comment by singe** [comment]
Can you post more information? Which website, which mana configuration? One thing to note, is that it will only work on sites which don't already have an HSTS entry e.g. if you go to encrypted.google.com it won't work, but if you go to another google site where the initial connection is HTTP it will.

**Comment by ra7bi** [comment]
if you type https in your browser manually then sslstrip will not work , because https will establish the handshake and encryption

**Comment by singe** [comment]
Yes. sslstrip prevents upgrades to SSL but can’t downgrade once its established. 

Also, mana toolkit is deprecated. Please see https://w1f1.net for the latest tools.
